gallery/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sonatype/react-shared-components-gallery",
-  "version": "2.5.1",
+  "version": "2.5.2",
```

Comment:
I feel like this creates changes that are visible enough that it warrants a minor version

Reply:
> I feel like this creates changes that are visible enough that it warrants a minor version

Ok: https://github.com/sonatype/sonatype-react-shared-components/pull/185/commits/2446d608fbf8db4370111dc18fffe7b11cef6cb3